ncm-sysconfig/src/main/pan/components/sysconfig/schema.pan

```diff
@@ -9,9 +9,7 @@ include 'quattor/schema';
 
 type component_sysconfig = {
     include structure_component
-    'files' ? string{}{}
+    'files' ? string(1..){}{}
```

Comment:
Can you add an annotation describing the structure that is required here in more detail? Also what version of panc does this require? Please call out in the commit log since I am not sure we have got around to deploying the latest release yet.

Reply:
Sure, I'll improve the docs.
It shouldn't need a recent pan compiler, string length validation has been around forever and the double nested dict comes from the original code from CERN.

Reply:
I could also break the type apart to make it clearer.